mailbox/backup/src/main/java/org/apache/james/mailbox/backup/zip/MailAccountZipIterator.java

```diff
@@ -134,6 +139,52 @@ private MailArchiveEntry fromMailboxAnnotationEntry(ZipEntryWithContent current,
         }
     }
 
+    private Optional<MailArchiveEntry> fromMessageEntry(ZipEntryWithContent entryWithContent) throws ZipException {
```

Comment:
we usually call `fromB` a factory method that belongs to a type A a create a A from a B.